src/components/save.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import Button from 'boundless-sdk/components/Button';
+import classNames from 'classnames';
+import enMessages from 'boundless-sdk/locale/en.js';
+import SaveIcon from 'material-ui/svg-icons/content/save';
+import {defineMessages, injectIntl, intlShape} from 'react-intl';
+import SaveDetailModal from './savedetailmodal';
```

Comment:
I guess for filenames we should either do everything lower-case or camel case? Personally I like SaveDetailModal.js best.